proto/buf/alpha/registry/v1alpha1/labels.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+enum LabelNamespace {
+  LABEL_NAMESPACE_UNSPECIFIED = 0;
+  // This is for globs, '*', in requests. When a label is created
+  // it will be assigned to a specific namespace.
+  LABEL_NAMESPACE_ALL = 1;
```

Comment:
Can a label exist in an ALL namespace? If the other options are all concrete, I'd say stick with UNSPECIFIED for this use-case.

Reply:
A label cannot/should not be created in the `ALL` namespace. Yeah, I was a bit concerned that this could cause some bugs. At the data model layer, this option won't exist, but having it at the API layer _might_ be confusing.

Reply:
I wouldn't expect a creation call to be made on ALL. Querying, though, could be a search for the name in all namespaces.